docs/Vector-Similarity-Hybrid-Queries.ipynb

```diff
@@ -0,0 +1,363 @@
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Collecting git+https://github.com/RediSearch/redisearch-py.git@params\n",
+      "  Cloning https://github.com/RediSearch/redisearch-py.git (to revision params) to /tmp/pip-req-build-xmid8x85\n",
+      "  Running command git clone --filter=blob:none --quiet https://github.com/RediSearch/redisearch-py.git /tmp/pip-req-build-xmid8x85\n",
+      "  Running command git checkout -b params --track origin/params\n",
+      "  Switched to a new branch 'params'\n",
+      "  Branch 'params' set up to track remote branch 'params' from 'origin'.\n",
+      "  Resolved https://github.com/RediSearch/redisearch-py.git to commit 1c938c33cf314c0403e487473f9022f645de35c8\n",
+      "  Installing build dependencies ... \u001b[?25ldone\n",
+      "\u001b[?25h  Getting requirements to build wheel ... \u001b[?25ldone\n",
+      "\u001b[?25h  Preparing metadata (pyproject.toml) ... \u001b[?25ldone\n",
+      "\u001b[?25hCollecting rmtest@ git+https://github.com/RedisLabs/rmtest@master\n",
+      "  Cloning https://github.com/RedisLabs/rmtest (to revision master) to /tmp/pip-install-3ygg7kn_/rmtest_44fbd4d60671454a8c306a47dd884045\n",
+      "  Running command git clone --filter=blob:none --quiet https://github.com/RedisLabs/rmtest /tmp/pip-install-3ygg7kn_/rmtest_44fbd4d60671454a8c306a47dd884045\n",
+      "  Resolved https://github.com/RedisLabs/rmtest to commit 4fb86bf3a287a1b0c5fc00f6d5bd8fb90ef8ba39\n",
+      "  Preparing metadata (setup.py) ... \u001b[?25ldone\n",
+      "\u001b[?25hCollecting redis==3.5.3\n",
+      "  Using cached redis-3.5.3-py2.py3-none-any.whl (72 kB)\n",
+      "Requirement already satisfied: six<2.0.0,>=1.16.0 in /home/alon/anaconda3/envs/vsim/lib/python3.8/site-packages (from redisearch==2.1.1) (1.16.0)\n",
+      "Requirement already satisfied: hiredis<3.0.0,>=2.0.0 in /home/alon/anaconda3/envs/vsim/lib/python3.8/site-packages (from redisearch==2.1.1) (2.0.0)\n",
+      "Requirement already satisfied: rejson<0.6.0,>=0.5.4 in /home/alon/anaconda3/envs/vsim/lib/python3.8/site-packages (from redisearch==2.1.1) (0.5.4)\n",
+      "Installing collected packages: redis\n",
+      "  Attempting uninstall: redis\n",
+      "    Found existing installation: redis 4.2.0rc1\n",
+      "    Uninstalling redis-4.2.0rc1:\n",
+      "      Successfully uninstalled redis-4.2.0rc1\n",
+      "\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
+      "rltest 0.5.0 requires redis<5.0.0,>=4.1.2, but you have redis 3.5.3 which is incompatible.\u001b[0m\u001b[31m\n",
+      "\u001b[0mSuccessfully installed redis-3.5.3\n",
+      "Requirement already satisfied: numpy in /home/alon/.local/lib/python3.8/site-packages (1.21.4)\n"
+     ]
+    }
+   ],
+   "source": [
+    "!pip install git+https://github.com/RediSearch/redisearch-py.git@params\n",
```

Comment:
you can use redis-py now
we will submit this PR now as-is and modify all ipynb files with new redis-py API after